fendermint/app/config/default.toml

```diff
@@ -138,6 +138,17 @@ host = "127.0.0.1"
 # JSON-RPC (POST) and WebSockets (GET) requests.
 port = 8545
 
+[eth.cors]
+# A list of origins a cross-domain request can be executed from
+# Default value '[]' disables cors support
+# Use '["*"]' to allow any origin
+allowed_origins = []
+# A list of methods the client is allowed to use with cross-domain requests
+# Suggested methods if allowing origins: "GET", "OPTIONS", "HEAD", "POST"
+allowed_methods = []
```

Comment:
I left these empty by default because there is no way to override them to be _empty_ using an env var (empty strings [are ignored](https://github.com/consensus-shipyard/ipc/pull/1021/files#diff-e8a390f79ef4d76cff941961d69b6cf5c6bc1c5a08a0f0f763c2eaab3029d953R326) by the config list parser).